.github/workflows/triage-issues.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow uses the following github action to automate
+# management of stale issues and prs in this repo:
+# https://github.com/marketplace/actions/close-stale-issues
+
+name: Close stale issues and PRs
+
+on: 
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * 0'
+  
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4.0.0
+        with:
+          days-before-issue-stale: 30
+          days-before-issue-close: 10
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
+          stale-issue-label: stale-no-activity
+          stale-issue-message: 👋 It looks like this issue has been open for 30 days with no activity. We'll mark this as stale for now, and wait 10 days for an update or for further comment before closing this issue out.
+          close-issue-message: This issue has been closed due to being inactive for 10 days with no further update.
```

Comment:
> being inactive for 10 days with no further update.

In this situation, the issue should be inactive for 40+ days (30 days before being marked as stale, and then 10 days until closing), right?

Also, (perhaps, other maintainers can come up with a better sentence but) using a bit more friendly message like "As this issue has been inactive for more than one month, let us close this. Thanks again for writing in here!" or so may be fine.

Reply:
How about: 
"As this issue has been inactive for more than one month, we will be closing it. Thanks again for writing in!"

Reply:
Thanks, looks great 👍